pinc/Round.inc

```diff
@@ -115,20 +115,6 @@ class Round extends Stage
         $this->honorifics = $honorifics;
         krsort($this->honorifics);
 
-        $this->project_unavailable_state = constant("PROJ_{$round_id}_UNAVAILABLE");
```

Comment:
Let's add a comment that project state attributes are defined later so that someone who sees an attribute being used on a Round object and looks here has a breadcrumb trail. eg:
```php
// The following project-state related attributes are created & set in project_states.inc
//     $this->project_unavailable_state
//     $this->project_waiting_state
//     $this->project_bad_state
//     $this->project_available_state
//     $this->project_complete_state
//     $this->project_states
```

Reply:
Done this.